engines/director/detection_tables.h

```diff
@@ -2311,6 +2339,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1_l("vcb", "", "Virtual Cocktail Bar", "f5277c53bacd27936158dd3867e587e2", 393446, Common::JA_JPN, 311),
 	MACDEMO1_l("vcb", "Demo", "VirtualCocktailBar Exective", "f5277c53bacd27936158dd3867e587e2", 394261, Common::JA_JPN, 311),
 
+	MACGAME1_l("vcbe", "", "VirtualCocktailBarExecutive", "f5277c53bacd27936158dd3867e587e2", 394073, Common::JA_JPN, 311),
```

Comment:
It looks like the demo directly above this should also be `vcbe`. Could you please move that one down?

Reply:
Done.